### PR TITLE
Added Plane Guide.  

### PR DESCRIPTION
**Pull request by insominx**
Implements a basic plane guide (known as a stencil in code).  This provides a cleaner way for mimicking a whiteboard for usages simply beyond painting on a flat surface.

Consider the use-case as a replacement for a virtual-whiteboard to be used by a teacher.  In such cases the appearance of the underlying grid is important as it communicates the space that it spans.  With the cube the lines from the sides, top, and back are always in view and change dramatically based on the angle it is viewed from.  Even looking at it straight on doesn't work because we are viewing things in perspective.

### CODE REVIEW

File: Assets/Prefabs/Panels/GuideToolsPanel.prefab [hunk 1]
```diff
@@ -1212,5 +1234,129 @@ MonoBehaviour:
...
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 4099382765129129040}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 483352d8a6ca1b64999b70a7a617fc54, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  m_DescriptionType: 0
+  m_DescriptionYOffset: 0
+  m_DescriptionText: Cube Guide
```
Comment (mikeskydev) [inline Assets/Prefabs/Panels/GuideToolsPanel.prefab:1348]: Change name to Plane Guide

**Review by mikeskydev — CHANGES_REQUESTED** [review]
Excited to have this! I have a few fixes.

The plane stencil prefab doesn't appear to be referenced on the widget manager in sketch controls, preventing the guide spawning.

The tooltip still reads 'Cube Guide' when you hover over it.

After wiring up the prefab, there's also an issue when layering up brushes in the same location, see screenshot:
image

This behaviour doesn't happen on the cube guide.

File: Assets/Scripts/Widgets/PlaneStencil.cs [hunk 2]
```diff
@@ -0,0 +1,201 @@
+﻿// Copyright 2020 The Tilt Brush Authors
```
Comment (mikeskydev) [inline Assets/Scripts/Widgets/PlaneStencil.cs:1]: Please can this be attributed as such: `Copyright 2021 The Open Brush Authors`

Comment (mikeskydev) [reply Assets/Scripts/Widgets/PlaneStencil.cs]: Sorry to be a pain, it should be Open Brush not Tilt Brush! 😅

Comment (insominx) [reply Assets/Scripts/Widgets/PlaneStencil.cs]: No worries, it's one of my first PR's so I appreciate your patience.

**Comment by insominx** [comment]
> After wiring up the prefab, there's also an issue when layering up brushes in the same location, see screenshot:
> image
> 
> This behaviour doesn't happen on the cube guide.

I'm not sure the true cause of this but the last commit does restore functionality.  Somewhere, there is an implicit restriction that the value returned from `FindClosestPointOnSurface` not have a 0 coordinate.  If you could provide additional insight, I would very much like to know!

**Review by mikeskydev — APPROVED** [review]
LGTM! Thanks for processing these changes!

**Pull request merged**